qemu/tests/cfg/sev_dhcert_boot.cfg

```diff
@@ -0,0 +1,13 @@
+- sev_dhcert_boot:
+    type = sev_dhcert_boot
```

Comment:
and do we also want to add some 'well known' filters for sev, such as ovmf, q35?